libraries/src/application/Helper.php

```diff
@@ -7,14 +7,22 @@
  * @license     GNU General Public License version 2 or later; see LICENSE
  */
 
+namespace Joomla\Cms\Application;
+
 defined('JPATH_PLATFORM') or die;
 
+use \JFactory;
+use \JLog;
+use \JFilterOutput;
+use \JInstaller;
+use \JText;
+
 /**
  * Application helper functions
  *
  * @since  1.5
  */
-class JApplicationHelper
+class Helper
```

Comment:
Even though it's namespaced, I'd suggest not using vague class names like `Helper`.  Gets awkward when you start loading the application helper, component helper, and installer helper in one class, for example.